js-apps/meep-frontend/src/js/meep-constants.js

```diff
@@ -215,9 +215,9 @@ export const ELEMENT_TYPE_DC = 'DISTANT CLOUD';
 export const ELEMENT_TYPE_CN = 'CORE NETWORK';
 export const ELEMENT_TYPE_EDGE = 'EDGE';
 export const ELEMENT_TYPE_FOG = 'FOG';
-export const ELEMENT_TYPE_UE = 'UE';
+export const ELEMENT_TYPE_UE = 'TERMINAL';
```

Comment:
ELEMENT_TYPE_UE seems to be used while parsing a scenario in scenario-utils.js. This change may break that code. Have you validated that UEs appear correctly in the frontend map?

Reply:
Fixed the marker change issue